node/src/chain_spec/mod.rs

```diff
@@ -19,6 +19,7 @@ pub mod oak;
 #[cfg(feature = "turing-node")]
 pub mod turing;
 
+#[allow(dead_code)]
```

Comment:
Unrelated changes to clean up compiler warnings. This is not actually dead code, it is only present under some features. Rather than be pedantic about the features let's just suppress the warning.